package.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "vets-json-schema",
-  "version": "6.7.4",
+  "version": "7.0.0",
```

Comment:
The changes to the tests in this PR are significant, but are these breaking changes to the actual schemas that get consumed? If so, I think a major version bump makes sense. 

But if these changes are backwards compatible, then I think a major version bump might be unnecessary, and a minor/patch version bump might suffice.

Reply:
this is a breaking change for the 20-0996 schema, yes

Reply:
Cool. Thank you.